astroquery/sdss/core.py

```diff
@@ -102,9 +103,9 @@ def query_crossid_async(self, coordinates, obj_names=None,
         """
 
         if (not isinstance(coordinates, list) and
-                not isinstance(coordinates, Column) and
```

Comment:
keep this logical indent

Reply:
I have restored the original indentation.